.github/CODEOWNERS

```diff
@@ -875,10 +875,6 @@ packages/kbn-test-eui-helpers @elastic/kibana-visualizations
 x-pack/test/licensing_plugin/plugins/test_feature_usage @elastic/kibana-security
 packages/kbn-test-jest-helpers @elastic/kibana-operations @elastic/appex-qa
 packages/kbn-test-subj-selector @elastic/kibana-operations @elastic/appex-qa
-x-pack/test_serverless @elastic/appex-qa
-test @elastic/appex-qa
-x-pack/test @elastic/appex-qa
-x-pack/performance @elastic/appex-qa
```

Comment:
I think CI will re-add these because you are the owners of those "packages"

Maybe adding an override negating it?
Or setting those packages' owners as an empty array?

Reply:
Thanks for explanation, I will try to set  owners to empty array and see if that works.

If not working, I will investigate how to override negating it (did you try it before?)